### PR TITLE
continue the ansible_ai_connect transition

### PR DESCRIPTION
**Pull request by goneri**
Jira Issue: <https://issues.redhat.com/browse/AAP-19331>

Continue the transition to `ansible_ai_connect`.

See: https://github.com/ansible/ansible-ai-connect-service/pull/990
See: https://github.com/ansible/ansible-ai-connect-service/pull/883

- [x] This code change is ready for production on its own
- [ ] This code change requires the following considerations before going to production:

### CODE REVIEW

File: tools/configs/nginx-wisdom.conf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 upstream uwsgi {
-    server unix:///var/run/uwsgi/ansible_wisdom.sock;
+    server unix:///var/run/uwsgi/ansible_ai_connect.sock;
```
Comment (manstis) [inline tools/configs/nginx-wisdom.conf:2]: Does [this](https://github.com/ansible/ansible-ai-connect-service/blob/main/tools/configs/uwsgi.ini) need updating too? If so, there's an [equivalent](https://github.com/ansible/ansible-ai-connect-operator/blob/main/roles/model/templates/model.configmap.wisdom-service.yaml.j2) in the Operator code-base that would also need updating.

**Review by manstis — APPROVED** [review]
LGTM :+1: 

Just a small comment about another file not in this PR.

File: tools/configs/nginx-wisdom.conf (hunk 1 above)
Comment (goneri) [reply tools/configs/nginx-wisdom.conf:2]: Yes you're right. I will remove this change from the PR.

**Review by manstis — DISMISSED** [review]
LGTM :+1:

File: tools/configs/uwsgi.ini [hunk 2]
```diff
@@ -14,7 +14,7 @@ lazy-apps = true                     ; if this is disabled, we must initialize t
                                      ; and feature_flags.py
 skip-atexit = true                   ; to avoid exception during the on_exit (e.g Pytorch)
 manage-script-name = true
-mount = /=ansible_wisdom.main.wsgi:application
+mount = /=ansible_ai_connect.main.wsgi:application
```
Comment (manstis) [inline tools/configs/uwsgi.ini:17]: Can you please also submit a PR for the Operator.

https://github.com/ansible/ansible-ai-connect-operator/blob/main/roles/model/templates/model.configmap.wisdom-service.yaml.j2#L28

**Review by manstis — APPROVED** [review]
LGTM :+1:

File: tools/configs/uwsgi.ini (hunk 2 above)
Comment (goneri) [reply tools/configs/uwsgi.ini:17]: https://github.com/ansible/ansible-ai-connect-operator/pull/214

**Pull request merged**